packages/documentation-site/patternfly-docs/pages/home.css

```diff
@@ -221,3 +137,10 @@
 .fadeInFour {
   animation-delay: 1.4s;
 }
+
+.pf-v5-c-page {
```

Comment:
This selector needs to be scoped to the home page. With it unscoped, it will apply to all page sections in the docs site. Note the misalignment from the extra padding

<img width="661" alt="Screenshot 2023-06-23 at 6 07 45 PM" src="https://github.com/patternfly/patternfly-org/assets/35148959/042b583d-4395-40cb-a4b3-53efe0e524f3">